DFC.Composite.Shell.Services/Neo4J/Neo4jService.cs

```diff
@@ -0,0 +1,99 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Options;
+using Neo4j.Driver;
+using System;
+using System.Threading.Tasks;
+using UAParser;
+
+namespace DFC.Composite.Shell.Services.Neo4J
+{
+    public class Neo4JService : INeo4JService
```

Comment:
You'd be better off using the existing Neo4j [nuget](https://www.nuget.org/packages/DFC.ServiceTaxonomy.Neo4j/) ([source](https://github.com/SkillsFundingAgency/dfc-servicetaxonomy-editor/tree/master/DFC.ServiceTaxonomy.Neo4j)), so that you get support for multi-graph, and also get the benefit of the testing it's already received (including unit and integration tests).

Reply:
Looking at this now